app/views/questions/show.html.erb

```diff
@@ -47,7 +47,27 @@
               <img src="https://kitt.lewagon.com/placeholder/users/random" class="avatar card-avatar">
               <div class="header-textbox">
                 <p class="user"><%= reply.user.email %></p>
-                <p class="created_at gray"><%= reply.created_at %> </p>
+                <p class="created_at gray"><%= reply.created_at.strftime("Replied on: %d/%m/%Y") %> </p>
+                <div class="rating-stars">
+                  <%# Add Average Rating for this user %>
+                  <span class="reviews-text">Reviews:</span>
+                  <% ratingsum = 0  %>
+                  <% count = 0 %>
+                  <% @reviews.each do |review| %>
+                    <% if review.user_id == reply.user_id %>
+                      <% ratingsum += review.rating %>
+                      <% count += 1 %>
+                    <% end %>
+                  <% end %>
+                  <% num = (ratingsum / count).round(0) %>
```

Comment:
you could get rid of the count by calling .length() on your @reviews.reject (or better, you could loop only trough the ones you use. I guess it would be faster)